lib/app_profiler.rb

```diff
@@ -60,17 +63,75 @@ module Viewer
   mattr_reader :after_process_queue, default: nil
 
   class << self
-    def run(*args, &block)
-      Profiler.run(*args, &block)
+    def run(*args, with_backend: nil, **kwargs, &block)
```

Comment:
with_backend optional keyword argument allows for the backend to be changed from the default for just one profiling session.

Reply:
```suggestion
    def run(*args, backend: nil, **kwargs, &block)
```